glean-core/tests/ping.rs

```diff
@@ -55,7 +55,14 @@ fn disabling_upload_clears_pending_pings() {
     glean.set_upload_enabled(false);
     assert_eq!(0, get_queued_pings(glean.get_data_path()).unwrap().len());
     // Disabling upload generates a deletion ping
-    assert_eq!(1, get_deletion_pings(glean.get_data_path()).unwrap().len());
+    let dpings = get_deletion_pings(glean.get_data_path()).unwrap();
+    assert_eq!(1, dpings.len());
+    assert_eq!(
+        "set_upload_enabled",
+        dpings[0].1["ping_info"].as_object().unwrap()["reason"]
```

Comment:
Can we pull this out into a variable? I find this `.1[...]` inline harder to parse.

```rust
let payload = &dpings[0].1;
```